engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -51,26 +81,45 @@ object WorkflowExecutionActor {
 final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescriptor: EngineWorkflowDescriptor) extends LoggingFSM[WorkflowExecutionActorState, WorkflowExecutionActorData] {
 
   val tag = self.path.name
-  startWith(WorkflowExecutionPendingState, WorkflowExecutionActorData())
 
-  def startJob(call: Call) = {
+  val calls = workflowDescriptor.backendDescriptor.workflowNamespace.workflow.calls
+  val callGraph = createCallGraph(calls)
+  val orderedGraph: List[List[Scope]] = Graph.topologicalSort(callGraph)
+
+  private def createCallGraph(calls: Seq[Scope]): Graph[wdl4s.Scope] = {
+    val callGraph = new Graph[wdl4s.Scope]
+    calls foreach callGraph.addNode
+    calls foreach { call =>
+      call.prerequisiteCalls foreach (callGraph.addEdge(call, _))
+    }
+    callGraph
+  }
+
+  private def startJob(call: Call) = {
     // TODO: Support indexes and retries:
+    log.info(s"Starting Call: [${call.fullyQualifiedName}]")
     val jobKey = BackendJobDescriptorKey(call, None, 1)
     val jobDescriptor: BackendJobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, Map.empty)
```

Comment:
This `Map.empty` was for a symbol map..which currently I'm not sure how to populate or how it should look like 😕